### ISSUE TITLE
UX and design implementation feedback 

### ISSUE THREAD

**Issue by linnutee** [decred/decrediton#663]
**Interaction items**
- [ ] Decimal marks on sending. 
Upon entering an amount for sending "," (comma symbol) does not go through. Displays a changed amount w/o the decimal numbers on verification. 
Should either accept both period and comma symbols or display an error when using comma instead of changing the amount, as users may not notice it. 

- [ ] Pending transaction reversed (top to bottom) listing
New/pending transactions are listed in opposite direction to the recent transaction list. 

- [ ] Error notifications
The error messages light up with almost any interaction to the field. (E.g. clicking on away after entering destination address causes the amount ones to light up.) These seem to be too sensitive and frequent at the moment. Ideally the error messages would live in the notifications not appear on matters that aren't relevant. 
<img width="739" alt="screen shot 2017-09-14 at 02 52 06" src="https://user-images.githubusercontent.com/17774057/30406058-048d459a-98f8-11e7-8af3-5449e8046249.png">
<img width="613" alt="screen shot 2017-09-14 at 02 53 16" src="https://user-images.githubusercontent.com/17774057/30406062-0a5a1cb4-98f8-11e7-87cb-e260ec45b718.png">

**Visual items** 
- [ ] Disabled button state 
Button:
`  background: #E7EAED;
  box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.1);`
Type:
`color: #8997a5; `

<img width="310" alt="screen shot 2017-09-14 at 02 22 53" src="https://user-images.githubusercontent.com/17774057/30405290-a3a5bc2a-98f3-11e7-8a8c-09a35eab65a3.png">

- [ ] Fields 
At the moment the fields have turned out to be in two different styles, neither fully from /send.html or mac or a library native. Other wise it would be a smaller issue but through-out the app there this creates a lot of inconsistency. So should either push for one or the other for a temporary fix. 
- Drop-downs have a new look, not sure if they're native or from some library?
- Text fields look partially correct: don't have the different type settings (for numbers), blue selection appears around them when clicked on, have shorter length,
![screen shot 2017-09-14 at 02 27 08](https://user-images.githubusercontent.com/17774057/30405741-5d9ab336-98f6-11e7-9e4c-a5960d986878.png)

- [ ] Voting view
Needs more spacing between the Voting Preferences. 
Align Stakepool dropdown to right. 
<img width="1646" alt="screen shot 2017-09-14 at 02 43 32 copy" src="https://user-images.githubusercontent.com/17774057/30405902-4fab2778-98f7-11e7-83cb-fe27637df424.png">

**Comment by matheusd** [comment]
If no one fixes this issues before then, I'll look at them after work on #649

**Comment by matheusd** [comment]
Fixed these issues here (will push after PR #670 is merged)

> Decimal marks on sending.
> Voting view
> Disabled Button State

A few questions on the others:

> Pending transaction reversed (top to bottom) listing

Generated a pending transaction and it appeared on the top (the transactions are ordered from newest to oldest). How should this ordering be? Where was the pending transaction supposed to go?

![screenshot_2017-09-19_11-14-06](https://user-images.githubusercontent.com/1752493/30597028-274f7dd6-9d2c-11e7-98ea-d3dc2e8b12e7.png)

> Error notifications
Just to clarify, you mean that for this two specific errors (when the address is blank and the amount is empty) no error should be shown, is that it?

> Fields (drop down)
They are from a library, but all references are to the same component, so re-styling should be easy. Should the styling be done according to the first image?

**Comment by matheusd** [comment]
@linnutee, take a look at the new select proposal.

![new-select mp4](https://user-images.githubusercontent.com/1752493/30822323-b1b3f85e-a1fe-11e7-821e-976f6b2948f4.gif)

**Comment by linnutee** [comment]
Is there any way to get rid of that bounding box? 

<img width="890" alt="screen shot 2017-09-25 at 21 21 17" src="https://user-images.githubusercontent.com/17774057/30824229-8511da3a-a237-11e7-982f-fa605bda49de.png">

**Comment by matheusd** [comment]
It is.
![new-select2 mp4](https://user-images.githubusercontent.com/1752493/30828459-1fe815f2-a214-11e7-90a4-f5a50d380a1a.gif)

**Comment by matheusd** [comment]
I changed all instances of the select component, only one left is the one from the seed. See if you have any comments or if I should leave it with the old style.
![screenshot_2017-09-26_11-45-59](https://user-images.githubusercontent.com/1752493/30871163-446d2068-a2bc-11e7-9353-065a1b3d760d.png)

**Comment by matheusd** [comment]
Hey, sorry to ping you @linnutee but do you have any thoughts on the above?

**Comment by linnutee** [comment]
Yep pardon my delayed response! These work nice and I think solve the inconsistency issue. 

Overall the confirm seed style looks relevant atm and we can improve that later on (didn't have the specifics for the word bits included in visual design). Only thing i'd remove is the arrow from right side and make the field either as correct size or add dragging corner to change height. (Like in github write field)

<img width="200" alt="screen shot 2017-09-29 at 23 23 57" src="https://user-images.githubusercontent.com/17774057/31034560-4f39ef2c-a56d-11e7-8277-7204646149d8.png">

![sep-29-2017 23-22-25](https://user-images.githubusercontent.com/17774057/31034607-7420fd6c-a56d-11e7-8d92-d4975692940a.gif)

**Comment by matheusd** [comment]
Ok, thanks for the feedback! I'll incorporate these changes then send a new PR.
